ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/EquipmentUpgradePlannerWindow.xaml

```diff
@@ -447,15 +448,22 @@
 					</DataTemplate>
 				</ItemsControl.ItemTemplate>
 			</ItemsControl>
+
+			<Border
+				Margin="2"
+				BorderBrush="Gray"
+				BorderThickness="1"
+				/>
+
+			<pager:PagingControlView Margin="5 0 0 0" ViewModel="{Binding PlannedUpgradesPager}" />
 		</StackPanel>
```

Comment:
Might need to change this to `WrapPanel`. The paging controls are getting cut off on half screen.

Reply:
Changed to WrapPanel